src/backend/effects/builtin/chat.js

```diff
@@ -55,7 +59,10 @@ const effect = {
         } else if (trigger.type === EffectTrigger.EVENT) {
             messageId = trigger.metadata.eventData?.chatMessage?.id;
         }
-
+        const meCommand = "/me ";
+        if (effect.me) {
+            effect.message = meCommand.concat(effect.message);
+        }
```

Comment:
We could use string template syntax here to avoid having to define a variable, like so:
```js
if (effect.me) {
    effect.message = `/me ${effect.message}`;
}
```

Reply:
Thank you! I simplified the if logic and used the string template syntax